src/parse-stream.js

```diff
@@ -325,6 +325,19 @@ export default class ParseStream extends Stream {
       });
       return;
     }
+    match = (/^#EXT-X-PROGRAM-DATE-TIME:?(.*)$/).exec(line);
+    if (match) {
+      event = {
+        type: 'tag',
+        tagType: 'date-time'
+      };
+      if (match[1]) {
+        event.dateTimeString = match[1];
+        event.dateTimeObject = new Date(match[1]);
```

Comment:
Now that you are using `new Date()` we do have timezone info right?

I that case maybe the string value doesn't matter.

Also, you now don't check if parsing succeeded.